_profiles/rayna_harris.md

```diff
@@ -0,0 +1,27 @@
+---
+# Required:
+title: "Rayna Harris"     # first (middle) last
+country:   US            # your country of residence (2 capital letters, e.g. US, GB, DE)
+
+# Optional (please add github or gravatar for your picture)
+github:    raynamharris   # your github ID (not full url)
+gravatar:     # your gravatar ID (the hex hash of your email, something like 123ef...123)
+homepage:  https://www.raynamharris.com/   # your personal homepage (full url)
+twitter:   raynamharris   # your twitter ID (not full URL, no leading '@')
+gitlab:       # your gitlab ID (not full URL)
+bitbucket:    # your bitbucket ID (not full URL)
+orcid:    0000-0002-7943-5650    # your orcid ID (not full URL)
+linkedin:  rayna-harris-484802ba   # your linkedin ID (not full url, i.e. the last bit of the url to your profile)
+email: rmharris@ucdavis.edu
+
+# Training WG
+training_roles: [instructor]              # subset of [facilitator, instructor, mentor], can stay empty ([])
+training_years: [2022]              # in which years did you help out? (e.g. [2020, 2019])
+
+# Don't modify the following setting
+layout: educator
+---
+
+<!-- Optional: Write something about yourself below the '- - >'.
+Hi! I'm Rayna. I'm an instructor with the [Carpentries](https://carpentries.org/). 
```

Comment:
Hi Rayna! Thanks for adding yourself.

This line needs to go after the `-->` though, i.e. at the end of the file :)   (it's an HTML style comment)

Reply:
ah thanks